### PR TITLE
Splash page

### PR DESCRIPTION
**Pull request by ellenwyllie**

### CODE REVIEW

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/index.html [hunk 1]
```diff
@@ -3,354 +3,767 @@
...
+    <section class="hero" data-externalizedAriaLabel="WELCOME_SECTION">
+		<div class="inner-hero">
+			<span id="star1" class="star"></span>
+			<span id="star2" class="star"></span>
+			<span id="star3" class="star"></span>
+			<span id="star4" class="star"></span>
+			<span id="star5" class="star"></span>
+			<div class="inner-hero__message row" data-externalizedAriaLabel="WELCOME_SECTION_CONTENT">
+				<div class="column">
+					<h3>Welcome to</h3>
```
Comment (aknguyen7) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/index.html]: Please update this to use externalize string

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/index.html [hunk 2]
```diff
@@ -3,354 +3,767 @@
...
+					<div id="inner-hero__logo">
+						<h1 id="inner-hero__title">Open Liberty</h1>
+						<p id="inner-hero__open_source">An IBM Open Source Project</p>
+					</div>
+					<h2 id="inner-hero__subtitle">A lightweight open framework for building fast and efficient cloud-native Java microservices.</h2>
```
Comment (aknguyen7) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/index.html]: use externalize string

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/index.html [hunk 3]
```diff
@@ -3,354 +3,767 @@
...
+				<div class="column img_container">
+					<img class="inner-hero__beam" src="/img/beam.png" alt="ufo and beam">
+				</div>
+			</div>
+			<img class="inner-hero__clouds" src="/img/clouds.png" alt="clouds">
+		</div>
+	</section>
+
+	<section class="logos">
+		<h2 class="logos__title">Open Liberty integrates with the Open Source community</h2>
```
Comment (aknguyen7) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/index.html]: use externalize string

**Comment by aknguyen7** [comment]
the changes to externalize string looks good. When I test the visual site, I saw the close banner button is kinda off when you have wide screen. Also it's not in full size when I simulate in phone/table mode

**Comment by aknguyen7** [comment]
#build

**Comment by aknguyen7** [comment]
#build

**Comment by aknguyen7** [comment]
#build

**Comment by mbroz2** [comment]
#run-libby-bot

**Pull request merged**